lib/github.rb

```diff
@@ -88,7 +88,7 @@ def get_prs(repo_url)
     return pr_info
   end
 
-  def group_project(cohort_students, data, repo)
+  def group_project(cohort_students, data, repo, cohort_name)
     students = []
     url = contributors_url(data)
```

Comment:
I know you didn't write this originally, but I really don't like the parameter name `data`. Something like `github_pr_data` might be better, if you feel like making the change.